fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java

```diff
@@ -639,6 +639,15 @@ protected int getRunningTxnNums() {
         return runningTxnNums;
     }
 
+    public long getRunningTxnNumsWithLock() {
```

Comment:
Using locks to protect primitive types is too expensive, you can use AtomicInteger, or no locks at all, you don't need to be very precise here.